src/cameras/PerspectiveCamera.js

```diff
@@ -135,6 +151,18 @@ THREE.PerspectiveCamera.prototype.copy = function ( source ) {
 	this.near = source.near;
 	this.far = source.far;
 
+	this.zoom = source.zoom;
+	this.focalLength = source.focalLength;
+
+	this.skewSlopeX = source.skewSlopeX;
+	this.skewSlopeY = source.skewSlopeY;
+
+	if ( source.view !== null && source.view !== undefined ) {
```

Comment:
> Which loader?

Oh great! `ObjectLoader` doesn't support any of the newer properties, although they are present in `toJSON`... 

"The loader" refers to `JSON.parse` then, I guess :-).